docs/DeveloperGuide.adoc

```diff
@@ -279,75 +279,185 @@ Refer to the guide <<DevOps#, here>>.
+|        3 |          3 | Health-focused user                               | add nutrition facts                                                              | I can see how much sugar, salt, fat etc is in the dish prepared
+|        4 |          3 | User who is more familiar with the metric system  | Convert between metric and imperial sizes.                                       | I can use the tools I have without needing to convert elsewhere
+|        4 |          3 | User who usually prepares multiple dishes as sets | group dishes into sets                                                           | I can be more organised when cooking
+|        4 |          3 | User who is inexperienced with software           | view the recipe in a GUI                                                         | I have more visual feedback to work with
+|        4 |          4 | User who is experienced with the software         | use shorthand commands                                                           | I can navigate the software more efficiently
+|        5 |          4 | User who wants to challenge myself                | have a timer/stopwatch                                                           | time myself when I cook dishes and have a "best time" feature
+|        5 |          5 | User who doesn't like screens and prefers paper   | print recipes as pdf/paper                                                       | I can refer to it more easily
+|        5 |          5 | User who likes sharing my cooking                 | post my recipes and dishes on social media                                       | share recipes and images for others to use
+|        5 |          5 | User who appreciates efficiency                   | add a recipe directly from online                                                | I can be efficient
+|===
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is `CookBuddy` and the *Actor* is the `User`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: List recipes
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  `User` requests to list recipes
+2.  `CookBuddy` displays the list of recipes
 +
 Use case ends.
 
 *Extensions*
 
+[none]
+* 1a. The name cannot be found, or the index is invalid.
++
+[none]
+** 1a1. `CookBuddy` throws an error message.
++
+Use case resumes at step 1.
+
 [none]
 * 2a. The list is empty.
 +
+[none]
+** 2a1. `CookBuddy` displays a message stating the list is empty
++
+Use case ends.
+
+
+
+[discrete]
+=== Use case: Delete recipe
+
+*MSS*
+
+1.  `User` requests to delete a specific _recipe_ by specifying its name or index
```

Comment:
Is this assuming the index of the recipe is already known? Perhaps you can list the recipes as part of MSS.